### ISSUE TITLE
alertIsPresent broken in W3C mode

### ISSUE THREAD

**Issue by sjerdo** [php-webdriver/php-webdriver#790]
### What are you trying to achieve? (Expected behavior)
Wait for a popup to open

### What do you get instead? (Actual behavior)
NoSuchAlertException

### How could the issue be reproduced? (Steps to reproduce)
```php
                $webdriver->wait(5, 200)->until(
                    WebDriverExpectedCondition::alertIsPresent()
                );
```

### Details

* Php-webdriver version:  1.8.2
* PHP version: 7.3.12
* Selenium server version: 
* Operating system: 
* Browser used + version: ChromeDriver (v81.0.4044.138)

**Comment by sjerdo** [comment]
alertIsPresent catches the wrong exception since this change:
https://github.com/php-webdriver/php-webdriver/commit/574684f29b8d513ae213c4567f2f71da92734db7

**Comment by OndraM** [comment]
Hi @sjerdo and thanks for the report!

I can confirm this behavior for W3C protocol, the exceptions are indeed swapped. I'll prepare a fix and improve tests.

**Comment by OndraM** [comment]
The fix is prepared in #792. :rocket:

**Comment by OndraM** [comment]
The fix has been merged to master and will be part of next php-webdriver release.

Thanks @sjerdo for the report!
